src/Main.java

```diff
@@ -2,14 +2,169 @@
+        //Объявите переменную clientOS, которая равна 0 или 1 (0 — iOS, 1 — Android).
+
+        System.out.println("Задача 1.");
+        int clientOS = (int) (Math.random() * 2);
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке.");
+        } else{
+            System.out.println("Установите версию приложения для Android по ссылке.");
+        }
+        System.out.println();
+
+        //Задача 2
+        //Усложним предыдущую задачу. Теперь нам нужно знать не только операционную систему телефона, но и год его создания.
+        //Ваша задача — написать программу, которая выдает соответствующее сообщение клиенту при наличии двух условий.
+        //Если год выпуска ранее 2015 года, то к сообщению об установке нужно добавить информацию об облегченной версии:
+        //Для iOS оно будет звучать так: «Установите облегченную версию приложения для iOS по ссылке».
+        //Для Android: «Установите облегченную версию приложения для Android по ссылке».
+        //Для пользователей телефонов 2015 года выпуска и позже нужно вывести обычное предложение об установке приложения.
+        //Для года создания телефона используйте переменную clientDeviceYear, в которой необходимо указать 2015 год.
+
+        System.out.println("Задача 2.");
+
+        int minYear = 2010;
+        int maxYear = 2025;
+
+        int clientDeviceYear = (int) (Math.random() * (maxYear - minYear + 1) + minYear);
+        if (clientDeviceYear < 2015){
+            if (clientOS == 0){
+                System.out.println("Установите облегченную версию приложения для iOS по ссылке.");
+            }else {
+                System.out.println("Установите облегченную версию приложения для Android по ссылке.");
+            }
+        }else {
+            if (clientOS == 0){
+                System.out.println("Установите версию приложения для iOS по ссылке.");
+            }else {
+                System.out.println("Установите версию приложения для Android по ссылке.");
+            }
+        }
+
+
+
+        System.out.println();
+
+        //Задача 3
+        //Напишите программу, которая определяет, является ли год високосным или нет.
+        //Переменную года назовите year, в которую можно подставить значение интересующего нас года. Например, 2021.
+        //Программа должна определять, високосный год или нет, и выводить соответствующее сообщение: « …. год является високосным» или «... год не является високосным».
+        //Небольшая справка: високосным является каждый четвертый год, но не является каждый сотый.
+        // Также високосным является каждый четырехсотый год. Год должен быть больше, чем 1584 (в котором был введен високосный год).
+
+        System.out.println("Задача 3.");
+        minYear = 1584;
+        int year = (int) (Math.random() * (maxYear - minYear + 1) + minYear);
+
+        if ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0) {
```

Comment:
```suggestion
        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```
скобки можно не писать, т.к. у И приоритет и так выше, чем у ИЛИ